.changeset/config.json

```diff
@@ -16,6 +16,9 @@
     "next-example",
     "vue-example",
     "amplify-ui-angular-mono",
-    "e2e"
+    "e2e",
+    "@aws-amplify/ui-angular",
+    "@aws-amplify/ui-react",
+    "@aws-amplify/ui-vue"
```

Comment:
Ignoring these packages so that we only publish `@aws-amplify/ui-svelte` from this branch.